src/tbb/resource.rc

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) 2005-2022 Intel Corporation
```

Comment:
If this is a new file why did you set the year range here 2005-2022? 
For the new file the rule is set the current year.